SyncUp.Data/Services/ImageService.cs

```diff
@@ -6,28 +6,25 @@
 using IntelliTect.SyncUp.Data;
 using SyncUp.Data.Models;
 using System.Net;
+using File = IntelliTect.Coalesce.Models.File;
 
 namespace IntelliTect.SyncUp.Data.Services;
-[Coalesce, Service]
```

Comment:
Removed the  coalesce service attribute because this never actually linked an image up with a tenant ( and it probably shouldn't, because we want to use this for groups, as well, and likely other models like comments... posts... etc)

This should just be an internal, non coalesce, service

Reply:
Not sure I fully undertstand...